pyoctoprintapi/octoprint_client.py

```diff
@@ -98,4 +100,34 @@ async def pause_job(self) -> None:
 
     async def resume_job(self) -> None:
         _LOGGER.debug("Sending resume job command")
-        await self._api.issue_job_command(JOB_COMMAND_PAUSE, JOB_COMMAND_PAUSE_RESUME)
\ No newline at end of file
+        await self._api.issue_job_command(JOB_COMMAND_PAUSE, JOB_COMMAND_PAUSE_RESUME)
+
+    async def disconnect(self) -> None:
+        _LOGGER.debug("Disconnecting from printer")
+        await self._api.issue_connection_command("disconnect")
```

Comment:
Can the command string be moved into the consts.py file?
```suggestion
        await self._api.issue_connection_command(CONNECTION_COMMAND_DISCONNECT)
```